contributing/releasing.md

```diff
@@ -231,6 +231,9 @@ synchronization within Google.
 
     git push origin stable develop
 
+You can now sync to the desired stable release. [go/mdc-releasing#re-run-the-import-script-against-githubstable](http://go/mdc-releasing#re-run-the-import-script-against-githubstable). Once you've submitted
+the internal CL, continue below to tag and publish the release.
+
 ## Publish the official release
 
 > Have all release-blocking clients given the go-ahead? **Do not create the official release
```

Comment:
Why would we need "release-blocking clients" to give a "go-ahead" if the internal CL is already submitted above?

Reply:
The change being reverted shouldn’t have been made in the first place. This revert gets us back to where we were. We can then iterate forward. As a general rule we want to keep commits focused on one thing at a time.

Reply:
@romoore as a general rule, reverts (or rollbacks) should be "pure" and only undo the original commit that they're targeting. When things are complicated it's the safest bet, instead of mixing in unknown new changes (or roll-forwards). Jeff's just following the general rule here, even though it's just documentation.